actions/transactions/routes.go

```diff
@@ -12,6 +12,29 @@ type Action struct {
 	actions.Action
 }
 
+// NewTransactionsHandler creates the specific package routes in restfull style
+func NewTransactionsHandler(appConfig *config.AppConfig, services *config.AppServices) *routes.Handler {
+	action := &Action{actions.Action{AppConfig: appConfig, Services: services}}
+
+	handler := routes.Handler{
+		BasicEndpointsFunc: routes.BasicEndpointsFunc(func(router *gin.RouterGroup) {
```

Comment:
I guess these routes should also be in the namepace `/api`. We need to redo in the same way as `OldAPIEndpointsFunc`

Reply:
I hope it's resolved - please take a look